frontend/packages/noobaa-storage-plugin/src/utils.ts

```diff
@@ -19,16 +19,14 @@ export type PrometheusMetricResult = {
 };
 
 export const getOBCPhase = (obc: K8sResourceKind): string => {
-  const phase: string = _.get(obc, 'status.Phase');
-  return phase ? phase.charAt(0).toUpperCase() + phase.substring(1) : 'Lost';
+  return _.get(obc, 'status.phase', 'Lost');
```

Comment:
```suggestion
  return _.get(obj, 'status.phase', 'Lost');
```
generalize the functions `getOBCPhase` and `getOBPhase`

Reply:
ack